spec.html

```diff
@@ -30822,6 +30837,9 @@ <h1>Static Semantics: Early Errors</h1>
             It is a Syntax Error if the List of Unicode code points that is SourceText of |LoneUnicodePropertyNameOrValue| is not identical to a List of Unicode code points that is a Unicode general category or general category alias listed in the &ldquo;Property value and aliases&rdquo; column of <emu-xref href="#table-unicode-general-category-values"></emu-xref>, nor a binary property or binary property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-binary-unicode-properties"></emu-xref>.
           </li>
         </ul>
+        <emu-note>
+          <p>The semantics are extended in <emu-xref href="#sec-patterns-static-semantics-early-errors-annexb"></emu-xref>.</p>
```

Comment:
I'd replaces "The semantics are extended" with "This section is amended" everywhere:

```suggestion
          <p>This section is amended in <emu-xref href="#sec-patterns-static-semantics-early-errors-annexb"></emu-xref>.</p>
```

I'd also probably move each occurrence to the top of its respective section to make it more likely to be seen.